civitai-downloader-v2/src/core/stream/streaming_search_engine.py

```
@@ -0,0 +1,335 @@
+#!/usr/bin/env python3
+"""
+Streaming Search Engine - ストリーム処理対応検索エンジン
+中間ファイルを使用したエラーリカバリ機能付き検索処理
+"""
+
+import asyncio
+import time
+from typing import Dict, Any, List, Optional, AsyncGenerator, Tuple
+import logging
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)